.hemtt/hooks/pre_build/01_set_version.rhai

```diff
@@ -0,0 +1,4 @@
+let modcpp = HEMTT_VFS.join("mod.cpp").open_file().read();
```

Comment:
`README.md` version won't be set, we should really set that and commit it before building a release though.

Reply:
@PabstMirror relevant for you mostly I guess.